gridstatus/ercot.py

```diff
@@ -3541,7 +3546,23 @@ def _handle_temperature_forecast_by_weather_zone_docs(
             # after the correction, the straight duplicate intervals remain, so we remove them
             df = df.drop_duplicates(subset=["Interval Start", "Publish Time"])
 
-        return df.sort_values("Interval Start")
+        # Raw data looks like
+        # DeliveryDate,HourEnding,Coast,East,FarWest,North,NorthCentral,SouthCentral,Southern,West,DSTFlag
+        # 11/02/2025,01:00,63.3, 58, 60, 57, 60, 63.5, 71.4, 59,N
+        # 11/02/2025,02:00,124.3, 113, 114, 111, 116.25, 120.5, 140.6, 112.8,N
+        # 11/02/2025,03:00,60.9, 55, 54, 54, 56.25, 58, 68.8, 54.2,Y
+        # 11/02/2025,03:00,60.9, 55, 54, 54, 56.25, 58, 68.8, 54.2,N
+        # The 3:00 ending hour is duplicated when it should be the 2:00 ending hour
```

Comment:
I checked against `ercot_load_forecast_by_forecast_zone`, turns out this dataset handles DST differently than than. maybe worth mentioning in this comment that this dataset handles it different than others from ERCOT. which is why we only have to fix it here

<img width="3450" height="368" alt="CleanShot 2025-10-26 at 20 24 34@2x" src="https://github.com/user-attachments/assets/79f5ba9b-4cc6-463b-b054-e5efc746ec3f" />